src/main/scala/com/fulcrumgenomics/vcf/HapCutToVcf.scala

```diff
@@ -73,19 +72,30 @@ import scala.collection.JavaConverters._
       |  3. The "MCL" tag gives the maximum change in likelihood if this SNP is made homozygous or removed.
       |  4. The "RMEC" tag gives the reduction in MEC score if we remove this variant altogether.
       |
+      |Additional FORMAT fields for phased variants are provided corresponding to per-genotype information produced by
+      |HAPCUT2:
+      |  1. The "PR" tag is 1 if HapCut2 pruned this variant, 0 otherwise.
+      |  2. The "SE" tag gives the confidence (log10) that there is not a switch error occurring immediately before the SNV
+      |  3. The "NE" tag gives the confidence (log10) that the SNV is not a mismatch (single SNV) error.
```

Comment:
I am using the description from @pjedge verbatim, but my interpretation would be that confidence that the call is correct.